pkg/api/routes.go

```diff
@@ -102,8 +106,9 @@ func (rh *RouteHandler) SetupRoutes() {
 			rh.blobLockWrapper(rh.DeleteBlobUpload)).Methods("DELETE")
 		g.HandleFunc("/_catalog",
 			rh.ListRepositories).Methods("GET")
-		g.HandleFunc("/",
-			rh.CheckVersionSupport).Methods("GET")
+		g.HandleFunc("/", rh.CheckVersionSupport).Methods("GET")
```

Comment:
don't change the indent here - it follows convention above

Reply:
Ok, I will follow that